packages/builders/src/interactions/contextMenuCommands/Assertions.ts

```diff
@@ -7,8 +7,7 @@ const namePredicate = s
 	.string()
 	.lengthGreaterThanOrEqual(1)
 	.lengthLessThanOrEqual(32)
-	// eslint-disable-next-line prefer-named-capture-group
-	.regex(/^( *[\p{P}\p{L}\p{N}\p{sc=Devanagari}\p{sc=Thai}]+ *)+$/u)
+	.regex(/.*\S.*/)
```

Comment:
```suggestion
	.regex(/\S/)
```
it suffices to match one character, the context is irrelevant